### PR TITLE
update ansible to publish license list to chef nodes.

### PR DESCRIPTION
**Pull request by npustchi**
Signed-off-by: npustchi <npustchi@bloomberg.net>

**Describe your changes**
We need to pass license list from inventory.yml in ansible to chef node. This will update it to chef.

**Testing performed**
Tested with jenkins for 1h1w toplogy.

### CODE REVIEW

File: ansible/playbooks/roles/common/filter_plugins/util.py [hunk 1]
```diff
@@ -62,6 +62,12 @@ def update_chef_node_host_vars(a, *args, **kw):
     node_details['normal']['host_vars'] = {}
     node_details['normal']['host_vars'].update({'interfaces': interfaces})
 
+    license = hostvars.get('license')
```
Comment (comay) [inline ansible/playbooks/roles/common/filter_plugins/util.py:65]: Shouldn't this actually be `licenses` given each host may have multiple licenses associated with it?

**Review by comay — COMMENTED** [review]
Also, I guess I expected there to be a `node_licenses()` function defined in `chef/cookbooks/bcpc/libraries/utils.rb`? Or is this going to be accessed in a different manner?

File: ansible/playbooks/roles/common/filter_plugins/util.py (hunk 1 above)
Comment (npustchi) [reply ansible/playbooks/roles/common/filter_plugins/util.py:65]: it is a type like in inventory we have run_list or transit.

**Comment by npustchi** [comment]
> Also, I guess I expected there to be a `node_licenses()` function defined in `chef/cookbooks/bcpc/libraries/utils.rb`? Or is this going to be accessed in a different manner?

there is :
https://github.com/bloomberg/chef-bcpc/blob/development/chef/cookbooks/bcpc/libraries/utils.rb#L136

**Review by comay — APPROVED** [review]

**Pull request merged**